docs/content/en/docs/implementing/slo.md

```diff
@@ -40,6 +40,12 @@ For more information,see:
 The Analysis result is exposed as an OpenTelemetry metric
 and can be displayed on dashboard tools, such as Grafana.
 
+> **Note** A preliminary release of the Keptn Analysis feature
+  is included in Keptn v0.8.3 and v0.9.0 but is hidden behind a feature flag.
+  See the
+  [Analysis](analysis.md/#differences-between-versions)
```

Comment:
I think the referenced `analysis.md` file is located in a different directory. Can you double check this link please?

Reply:
Whoops!  Careless copy/paste -- it's fixed now.